src/main/java/org/prebid/server/hooks/v1/Module.java

```diff
@@ -0,0 +1,13 @@
+package org.prebid.server.hooks.v1;
+
+import java.util.Set;
+
+/**
+ * Cares of the module identification among other modules and supplies a collection of available {@Hook}s.
+ */
+public interface Module {
+
+    String code();
```

Comment:
Is the `code` a unique identifier or can there be duplicates?

Reply:
```suggestion
    /**
     * A unique identifier that should be unique among other available modules.
     */
    String code();
```

Maybe document a few of the usages of this code? Analytic events, logging, referencing in other modules?

Reply:
Thanks for pointing, i'll just document what we imaging how it should work so far.